### PR TITLE
Beautify help command

### LINKED ISSUE #126: As a Customer Service Officer, I want to know the command format to enter
so that I can make lessons more convenient for tutors and students 
in the case they are unable to make it for a specific lesson

### PR DESCRIPTION
**Pull request by muhammad-khair**
### Work Done:  
- Add codecov badge in README.md
- Add help command summary table in HelpWindow (for now it is a screenshot)  
- Cleaned up header in MainWindow  
- Set text wrap for ResultDisplay to true  

image

image

Closes #126

### CODE REVIEW

File: src/main/resources/view/HelpWindow.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
 #copyButton, #helpMessage {
     -fx-font-family: "Open Sans";
 }
+
+#helpHeader {
+    -fx-font-family: "Open Sans Bold";
+    -fx-font-size: 22px;
```
Comment (michael-lee-sk) [inline src/main/resources/view/HelpWindow.css:7]: Would a bigger font size be better?

File: src/main/java/seedu/tuitione/ui/HelpWindow.java [hunk 2]
```diff
@@ -34,6 +45,8 @@
      */
     public HelpWindow(Stage root) {
         super(FXML, root);
+        helpHeader.setText(HELP_HEADER);
+        helpCommands.setImage(getImage(HELP_COMMAND_IMAGE));
```
Comment (michael-lee-sk) [inline src/main/java/seedu/tuitione/ui/HelpWindow.java:49]: Think the image you have added would be much more useful to a user looking for instructions! Good catch

**Review by michael-lee-sk — APPROVED** [review]
Overall looks fine, maybe only the portion abt the font size

**Comment by muhammad-khair** [comment]
### New updates:  
- Add lesson size to UI  

image

File: src/main/resources/view/HelpWindow.css (hunk 1 above)
Comment (muhammad-khair) [reply src/main/resources/view/HelpWindow.css:7]: Hmm sure thing!

**Pull request merged**